polyfills/Intl/DateTimeFormat/config.toml

```
@@ -0,0 +1,29 @@
+aliases = [
+  "Intl"
```

I think we would also want to alias the removed polyfill for `formatToParts` for those users which are requesting it, otherwise this would be a breaking change in the API.
```suggestion
  "Intl",
  "Intl.DateTimeFormat.prototype.formatToParts"
```